integration-test/src/it/nexus-staging-maven-plugin-removal-disabled/invoker.properties

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2012 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+invoker.goals=install
```

Comment:
Shouldn't be invoker goal be deploy to verify you can deploy with that plugin being there?

Reply:
Oh. Oops. Didn't read the rest of the test - you're verifying deployment failure? Actually, it should still be deploy but the invoker goal should fail?

Reply:
This test tests one case of the plugin not being removed in order to test the `-DnexusStagingMavenPluginRemoval=false` option. I verify that the plugin is still present in the pom. In that case, it _cannot_ deploy with the plugin in there. Basically, it's trying to test the default case that this PR is meant to fix.

I would like to test the deploy behavior in this case, but it fails so then the build fails. I guess I can improve this test by letting the deploy fail and then checking the `build.log` for the message?

Reply:
I suppose, its probably not that important .... we know its going to fail anyway. It is possible to 'expect' an invoker goal to fail - several tests do that already